src/models/GameFilter.py

```diff
@@ -19,5 +21,9 @@ def __post_init__(self):
             object.__setattr__(self, "playerrangetype", str(self.playerrangetype))
         if self.playstyle is not None:
             object.__setattr__(self, "playerstyle", str(self.playstyle))
+        if self.mincomplexity is not None:
+            object.__setattr__(self, "mincomplexity", float(self.mincomplexity))
```

Comment:
nit: if we're expecting that the complexity value will be in the range of 1 to 5 inclusive can we make this an `int` instead of a `float`? It would be easier to reason about integers versus floats when it comes to comparisons between the two.

Reply:
Ahhhh this is because complexity ratings usually have decimals, so a user may say they want a range 1.8-2.8. 

Although this makes me think that it isn't not obvious on the front end we expect decimals.  Going to add something to make it somewhat apparent